mods/weather/init.lua

```diff
@@ -56,16 +56,17 @@ minetest.register_globalstep(
 	    local weathertype = weather_pr:next(0, 19)
 
 	    -- on avg., every 1800 frames the weather.weather will change to one of:
-	    -- 13/20 chance of clear weather
-	    -- 6/20 chance or stormy weather
+	    -- 15/20 chance of clear weather
+	    -- 3/20 chance or stormy weather
 	    -- 1/20 chance of snowstorm weather
 
-	    if weathertype < 13 then
+	    if weathertype < 15 then
 	       weather.weather = "clear"
 	    elseif weathertype < 19 then
 	       weather.weather = "storm"
 	    elseif weathertype < 20 and snow_enable then
 	       weather.weather = "snowstorm"
+	    else weather.weather = "clear"
```

Comment:
Hi rubenwardy,

I'm a noob in github stuff, so I'm not sure if your comment is a good thing, a bad thing, a mistake that I've done or if I need to do something with my pull request.

Should I re add this new line again?

Thanks.